src/widget/form/genericchatform.cpp

```diff
@@ -199,6 +203,7 @@ GenericChatForm::GenericChatForm(QWidget *parent)
             this, &GenericChatForm::onChatContextMenuRequested);
 
     new QShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_L, this, SLOT(clearChatArea()));
+    new QShortcut(Qt::ALT + Qt::Key_Q, this, SLOT(quoteSelectedText()));
```

Comment:
@fazotron
>`CTRL` + `SHIFT` + `v` is a common shortcut to paste unformatted text in different software. I think it is a bad way to use it for another purposes.

 I see. On the other hand, using `ALT` + `q` to paste quote feels a bit weird to me.

Good to know that some older IMs used that :)

And yeah, in future it would be nice to have keyboard shortcuts configurable in settings.